appPostBoard.js

```diff
@@ -0,0 +1,173 @@
+// 'use strict'
+
+var table = document.getElementById('postBoard');
+var nameData;
+
+var jobNumber = 1;
+var trElement;
+var cityName;
+var cityMarker;
+var markerLabel;
+// assigning localstorage array to the new variable to use in rendering rows
+var clientArray = JSON.parse(localStorage.getItem('lsClientArray'));
+
+//console.log(clientArray);
+
+var citiesCoordinates = [
+  { name: 'seattle', lat: 47.6062, lng: -122.3321 },
+  { name: 'redmond', lat: 47.6740, lng: -122.1215 },
+  { name: 'bellevue', lat: 47.6101, lng: -122.2015 },
+  { name: 'sammamish', lat: 47.6163, lng: -122.0356 },
+  { name: 'kirkland', lat: 47.6769,  lng: -122.2060 },
+  { name: 'mercer island', lat: 7.5707, lng: -122.2221 },
+  { name: 'bothel', lat: 7.7610, lng: -122.2056 },
+  { name: 'new castle', lat: 47.5390, lng: -122.1557 },
+  { name: 'renton', lat: 47.4829, lng: -122.2171 },
+  { name: 'issaquah', lat: 47.5301, lng: -122.0326 },
+  { name: 'carnation', lat: 47.6479, lng: -121.914 },
+  { name:  'kent', lat: 47.3809, lng: -122.2348 },
+  {name: 'tukwila', lat:47.4740 , lng:-122.2610},
+  {name: 'woodinville', lat:47.7543 , lng: -122.1635},
+  {name: 'lynnwood', lat:47.8209 , lng:-122.3151 },
+  {name: 'tacoma', lat:47.2529 , lng:-122.4443},
+  {name: 'puyallup', lat: 47.1854, lng:-122.2929 },
+];
+var cityNamesArray = ['seattle', 'redmond', 'bellevue', 'sammamish', 'kirkland', 'mercer island',
+'bothel', 'new castle', 'renton', 'issaquah', 'carnation', 'kent','tukwila', 'woodinville', 'lynwood', 'tacoma', 'puyallup']
+;
+
+var clientEnteredCities = [];
+
+//console.log(clientArray);
+function renderHeader() {
+  trElement = document.createElement('thead');
```

Comment:
I was wondering how to do it in a cleaner way. Thank you! 
Wouldn't it be better to do the header in html instead?